astropy/utils/console.py

```diff
@@ -199,6 +199,8 @@ def terminal_size(file=None):
             if width is None:
                 width = 80
             return lines, width
+    else:
```

Comment:
I don't think this improves the code (though almost certainly it could do with a refactor).